backend/metamodel/syson-sysml-metamodel/src/main/java/org/eclipse/syson/sysml/impl/TypeImpl.java

```diff
@@ -183,8 +192,11 @@ public EList<Feature> getEndFeature() {
      */
     @Override
     public EList<Feature> getFeature() {
-        List<Feature> data = new ArrayList<>();
-        return new EcoreEList.UnmodifiableEList<>(this, SysmlPackage.eINSTANCE.getType_Feature(), data.size(), data.toArray());
+        Feature[] data = getFeatureMembership().stream()
```

Comment:
Could you rename this variable from `data` to `features` please?

Reply:
Done